set -u defined in test-driver propagates to test script which doesn't have
diff_opts initialized. test script fails with 'unbound variable' error.

This is Solaris specific patch till the fix gets to upstream.

--- a/examples/test	2020-09-21 07:00:47.000000000 +0200
+++ b/examples/test	2020-11-13 16:55:13.007708974 +0100
@@ -37,6 +37,7 @@
 # If diff supports --strip-trailing-cr, use it, to avoid EOL issues
 # when testing Java programs on Windows.
 echo "checking for diff --strip-trailing-cr..."
+diff_opts=
 if diff --strip-trailing-cr "$1" "$1"; then
   diff_opts=--strip-trailing-cr
 fi
